nuxeo-features/nuxeo-platform-mail/nuxeo-platform-mail-core/src/main/java/org/nuxeo/ecm/platform/mail/security/MailMessageSecurityPolicy.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *      André Justo
+ */
+package org.nuxeo.ecm.platform.mail.security;
+
+import static org.nuxeo.ecm.platform.mail.utils.MailCoreConstants.MAIL_MESSAGE_TYPE;
+
+import java.security.Principal;
+
+import org.nuxeo.ecm.core.api.security.ACP;
+import org.nuxeo.ecm.core.api.security.Access;
+import org.nuxeo.ecm.core.api.security.SecurityConstants;
+import org.nuxeo.ecm.core.model.Document;
+import org.nuxeo.ecm.core.query.sql.model.SQLQuery;
+import org.nuxeo.ecm.core.security.AbstractSecurityPolicy;
+
+/**
+ * Security policy that denies Write access on MailMessage documents.
+ *
+ * @since 10.1
+ */
+public class MailMessageSecurityPolicy extends AbstractSecurityPolicy {
```

Comment:
What's the purpose of having a test succeeding when the policy is not deployed? Since the policy is going to be always deployed, it's testing a use-case that never happens.

Reply:
It's showing that it's really the deployed policy that does this change of behavior, not some unrelated platform change.